proto/cosmos/bank/v1beta1/tx.proto

```diff
@@ -16,6 +16,9 @@ service Msg {
 
   // MultiSend defines a method for sending coins from some accounts to other accounts.
   rpc MultiSend(MsgMultiSend) returns (MsgMultiSendResponse);
+
+  // SetSendEnabled is a governance operation for setting the SendEnabled flag on any number of Denoms.
+  rpc SetSendEnabled(MsgSetSendEnabled) returns (MsgSetSendEnabledResponse);
```

Comment:
We shouldn't have this in this PR. Please make a seperate PR for governance operations to update these :)

Reply:
Is there a reason to split it out other than making a PR smaller? I ask because without this endpoint, there would be no way to set the `SendEnabled` state for denoms. Currently it's done through a governance proposal to set the bank module's Params. Having this is required in order to maintain current functionality.

Reply:
Sounds good. Give me a bit and I'll have it split out.

Reply:
Okay. I split it out the `MsgSetSendEnabled` stuff as PR #11981.

I also made a couple other small tweaks:
1. I default to using `nil` for the `Params.SendEnabled` slice (instead of an empty slice).
2. I updated the bank keeper's `SetParams` function to set the `SendEnabled` entries in the new way. That way, anyone using that method won't have weird things happen.